cmd/policy-assistant/pkg/matcher/explain.go

```diff
@@ -141,3 +140,14 @@ func PortMatcherTableLines(pm PortMatcher, kind PolicyKind) []string {
 		panic(errors.Errorf("invalid PortMatcher type %T", port))
 	}
 }
+
+func priorityTableLine(e Effect) string {
+	if e.PolicyKind == NetworkPolicyV1 {
+		return "All peers allowed"
```

Comment:
Since we're doing the same for (B)ANP, may help the reader to be explicit:
```suggestion
		return "NPv1: All peers allowed"
```